src/main/java/thestonedturtle/mahoganyhomes/MahoganyHomesOverlay.java

```diff
@@ -64,6 +64,49 @@ class MahoganyHomesOverlay extends OverlayPanel
 		getMenuEntries().add(new OverlayMenuEntry(RUNELITE_OVERLAY, CLEAR_OPTION, "Contract"));
 	}
 
+	// Adds a line to the panel describing a material. type is the
+	// un-pluralized name of the material, min and max are the range
+	// required, and have is the number in the player's inventory.
+	void addMaterialLine(String type, int min, int max, int have) {
```

Comment:
There are multiple checkstyle errors in this file.

To avoid these in the future, and see what to fix currently, you should add the `checkstyle.xml` file in the root directory to your intelij settings.

1) Settings (ctrl+alt+s)
2) Tools -> Checkstyle
3) Add the local checkstyle file and give it a name.

Ex:
![image](https://github.com/TheStonedTurtle/Mahogany-Homes/assets/29030969/788a07e8-c1ed-4768-9a54-c896798dfc57)